### DISCUSSION TITLE
Move GIScience/GH code to GIScience/openrouteservice as maven multiproject

### DISCUSSION THREAD

**Discussion by MichaelsJP** [GIScience/openrouteservice#1989] [Refactoring]
- move git history since GH 4.0 to repo as well

- [x] copy git history over
- [x] merge in a test branch
- [ ] construct as a multi project pom
- [ ] sort out dependency problems
- [x] fix unit tests for gh
- [ ] fix unit tetsts for ors
- [ ] Add gh apache license to files from the branch

**Comment by TheGreatRefrigerator** [comment]
we need to discuss this further once @takb and @aoles are back.
